src/styles/components/_content.scss

```diff
@@ -17,42 +17,6 @@
 	font-weight: $g-headings-font-weight;
 }
 
-.c-h1,
```

Comment:
Als we de styles bij de component zetten gaan we volgende scenario niet kunnen doen:
Ik heb een h1 nodig voor aan te duiden dat het de page titel is maar ik wil de c-h2 class gebruiken voor een kleinere font-size